framework/tlvf/AutoGenerated/src/tlvf/ieee_1905_1/tlvWscM2.cpp

```diff
@@ -454,7 +454,10 @@ bool tlvWscM2::add_encrypted_settings(std::shared_ptr<WSC::cWscAttrEncryptedSett
         TLVF_LOG(ERROR) << "No call to create_encrypted_settings was called before add_encrypted_settings";
         return false;
     }
-    if (ptr->getStartBuffPtr() != (uint8_t *)m_encrypted_settings) {
+    uint8_t *src = (uint8_t*) m_encrypted_settings;
+    if (!m_parse__) {
```

Comment:
empty if - again, think this case got messed up, This is a simple list of uint8_t inside a class

Reply:
The empty 'if' should be removed. Arnout already addressed the part in the python code that generates it.